src/test/java/uk/gov/hmcts/reform/managecase/security/JwtGrantedAuthoritiesConverterTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.managecase.security;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.oauth2.jwt.Jwt;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class JwtGrantedAuthoritiesConverterTest {
+
+    private static final String ACCESS_TOKEN = "access_token";
+
+    @Mock
+    private IdamRepository idamRepository;
+
+    @InjectMocks
+    private JwtGrantedAuthoritiesConverter converter;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    @DisplayName("No Claims should return empty authorities")
+    void shouldReturnEmptyAuthoritiesWhenClaimNotAvailable() {
+        Jwt jwt = mock(Jwt.class);
```

Comment:
We create mock in all test methods. Maybe we could move it to the class level?

Reply:
done